modules/ROOT/pages/register-server.adoc

```diff
@@ -3,20 +3,21 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-If you encounter an error when performing the steps in xref:runtime-manager::servers-create.adoc[Create a Server in Runtime Manager]:
+If you encounter a Mule runtime engine registration error when xref:runtime-manager::servers-create.adoc[creating a server in Runtime Manager], you may need to configure Mule to trust the TLS certificates you are using. 
 
-Before registering Mule runtime engine, you must configure Mule to trust the TLS certificates you configured:
+To do so:
+
+. Get the public certificate for Anypoint Platform Private Cloud Edition.
+. Add the certificate to the truststore of the JVM you are using for the Mule runtime engine.
+
+Using the following example, set `$ANYPOINT_DNS` to your DNS for Anypoint Platform Private Cloud Edition. Replace the keystore location and password for your JVM's truststore.
 
-.. Get the public certificate for Mule runtime engine.
-.. Add the certificate to the truststore of the JVM you are using for the Mule runtime engine.
-+
-For example:
-+
 ----
-$ keytool -printcert -sslserver $ANYPOINT_DNS:443 -rfc > /tmp/anypoint.crt
-$ keytool -importcert -alias anypoint.my-company -keystore /etc/ssl/certs/java/cacerts -storepass changeit -file /tmp/anypoint.crt
+$ keytool -printcert -sslserver $ANYPOINT_DNS:443 -rfc > /tmp/PCE_Certificate.cer
+$ keytool -importcert -alias anypoint.my-company -keystore <path to keystore> -storepass changeit -file /tmp/PCE_Certificate.cer
 ----
-+
-In the previous example, $ANYPOINT_DNS should be set to your DNS for Anypoint Platform Private Cloud Edition.
-Replace the location and password for your JVM's truststore.
+
+For more information about the CA cert truststore, refer to your JVM vendor's documentation. For example,  https://docs.oracle.com/en/java/javase/17/security/java-security-overview1.html#GUID-EB29B931-D1DE-420B-849C-D731A8DF9CDC[Oracle documentation^].
```

Comment:
I ran a search and did not see `certificate` abbreviated to `cert` except in the context of `cacert`?